extensions/tags/less/common/TagLabel.less

```diff
@@ -16,7 +16,7 @@
   }
 
   &.colored {
-    --tag-color: @body-bg;
+    --tag-color: var(--contrast-color, var(--body-bg));
```

Comment:
@SychO9 is this some kind of fallback logic?

Reply:
Yes, it uses the contrast color if it is set (like using the text contrast classes) otherwise it'll use he default value. In this case the default is `body-bg`

Reply:
That's awesome, I learn a lot with this refactor, thank you!